services/meta/client.go

```diff
@@ -957,35 +957,6 @@ func (c *Client) updateAuthCache() {
 	c.authCache = newCache
 }
 
-// snapshot will save the current meta data to disk
-func snapshot(path string, data *Data) error {
-	file := filepath.Join(path, metaFile)
-	tmpFile := file + "tmp"
-
-	f, err := os.Create(tmpFile)
-	if err != nil {
-		return err
-	}
-	defer f.Close()
-
-	var d []byte
-	if b, err := data.MarshalBinary(); err != nil {
-		return err
-	} else {
-		d = b
-	}
-
-	if _, err := f.Write(d); err != nil {
-		return err
-	}
-
-	if err = f.Close(); nil != err {
-		return err
-	}
-
-	return os.Rename(tmpFile, file)
```

Comment:
I think using `_unix.go` is better since another part of the codebase uses that. More consistency in the codebase is always a good thing.